website/blog/2024-02-13-run-mongodb-workloads-ferretdb-tembo.md

```diff
@@ -0,0 +1,228 @@
+---
+slug: run-mongodb-workloads-ferretdb-tembo
+title: 'How to Run MongoDB Workloads with FerretDB on Tembo'
+authors: [alex]
+description: >
+  In this blog post, we’ll show you how to run FerretDB on Tembo for your production workloads.
+image: /img/blog/ferretdb-tembo.jpg
+tags: [tutorial, postgresql tools, open source, cloud]
+---
+
+![How to Run MongoDB Workloads with FerretDB on Tembo](/img/blog/ferretdb-tembo.jpg)
+
+[FerretDB](https://www.ferretdb.com/) is on a mission to bring MongoDB workloads back to open source; we are adding MongoDB compatibility to relational DBs like [Postgres](https://www.postgresql.org/) and [SQLite](https://www.sqlite.org/).
```

Comment:
FerretDB is not on a mission to bring MongoDB _workloads_ back to open source. We are totally fine with FerretDB being used with proprietary applications.